how-tos/relpro.md

```diff
@@ -156,50 +156,87 @@ $ python releasetasks_graph_gen.py --release-runner-ini=../../../release-runner.
 $ python releasetasks_graph_gen.py --release-runner-ini=../../../release-runner.ini --branch-and-product-config=/home/cltbld/releasetasks/releasetasks/release_configs/prod_mozilla-esr52_firefox_rc_graph_2.yml --common-task-id=$TASK_TASKID_FROM_GRAPH1
 ```
 
+## 3. signoffs
```

Comment:
I think there are two different things happening in this section. I'm trying to add a section similar to the existing ones to make sure that this manual part of the release process (and how to handle it is documented). The "example" part is a very specific tutorial/documentation for "how to signoff", as you suggest. I can easily see that part living elsewhere in the doc.

For background, when I read this doc, it looked to me like the "actions" section was a list of things that humans need to do by hand for a release. If we were to remove the signoffs section altogether I'm concerned that it may be confusing how Release Promotion ever gets to the "publish release" section. I'll defer to whatever those who work with these docs more regularly think is best, though.

Thanks for the perspective on this!